Anime-App/Feature/Anime/AudioManger/AudioManager.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AudioManager.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 27/08/2023.
+//
+
+import Foundation
+import AVFoundation
+
+class AudioManager {
+    
+    private var name: String
+    private var player: AVAudioPlayer?
+
+    init(name: String) {
```

Comment:
So i would actually move this param to the play function, think about it if you want to play different sounds you'd have to create a new instance of this everytime. Instead if it was a function param then you could just call `play(name: "pop")` and then just play random sounds without needing to create a new instance. One of those things to think about when you architect software is how do i want to be able to use this?